build_system/src/build.rs

```diff
@@ -5,16 +5,18 @@ use std::ffi::OsStr;
 use std::fs;
 use std::path::Path;
 
+/// Represents build arguments.
```

Comment:
Please remove this comment as it is a duplicate of the code itself, so it doesn't bring anything more than the code.